src/common/config_opts.h

```diff
@@ -1416,6 +1416,8 @@ OPTION(rgw_swift_auth_entry, OPT_STR, "auth")  // entry point for which a url is
 OPTION(rgw_swift_tenant_name, OPT_STR, "")  // tenant name to use for swift access
 OPTION(rgw_swift_account_in_url, OPT_BOOL, false)  // assume that URL always contain the account (aka tenant) part
 OPTION(rgw_swift_enforce_content_length, OPT_BOOL, false)  // enforce generation of Content-Length even in cost of performance or scalability
+OPTION(rgw_swift_verify_object_acl, OPT_BOOL, true)  // if you want to be consistent with SWIFT, set false to skip and propose to handle
```

Comment:
@Jing-Scott (NIT) Please consider to keep PR title < 50 characters
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits